modules/nw-rfhe-creating_bmc_event_sub.adoc

```diff
@@ -0,0 +1,86 @@
+[id="subscribing-hw-events_{context}"]
+= Subscribing to Redfish BMC hardware events for a cluster node
+
+Users of bare metal hardware may want to receive events from the baseboard management controller (BMC) to act on them in case of a hardware fault, such as an increase in temperature or removal of a device. The Redfish standard includes the ability to subscribe to events, which will cause hardware events to be sent in a particular format to a destination URI. Configuring a subscription is done using the Metal3 API.
+
+Execute the following procedure from the desired node to configure event subscriptions and alerts.
+
+[IMPORTANT]
+====
+This feature only works on physical hardware running Redfish with support for events with a vendor interface set to `redfish` or `idrac-redfish`. 
+====
+
+.Procedure
+
+. Power off the bare metal node by using the baseboard management controller, and ensure it is off.
+
+. Retrieve the user name and password of the bare metal node's baseboard management controller. 
+
+.Confirm the hardware has EventService enabled by using the iDRAC/iLO interface [Add Steps] or the following cURL command:
```

Comment:
iDRAC/ILO consoles will be different across different hardware. I don't think we can cover all procedures. Could just refer to the iDRAC/iLO generally. The curl command should be sufficient, since the redfish API should be up regardless of the BMC type